apps/meteor/app/slashcommands-help/server/server.ts

```diff
@@ -17,55 +16,57 @@ interface IHelpCommand {
 	command: string;
 }
 
-function Help(_command: 'help', _params: string, item: IMessage): void {
-	const userId = Meteor.userId() as string;
-	const user = Users.findOneById(userId);
+slashCommands.add(
+	'help',
+	function Help(_command, _params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.